File: week 0/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get the height
    int height = 0;
    do
    {
        height = get_int("Height: ");
    }
    while (!(height >= 1 && height <= 8));

    // Loop through rows
    for (int row = 0; row < height; row++)
    {
        // Print the left half of the pyramid
        for (int col = 0; col < height; col++)
        {
            if (col >= height - row - 1)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        // Print the gap
        printf("  ");

        // And the right half of the pyramid
        for (int col = 0; col <= row; col++)
        {
            printf("#");
        }

        // Then newline
        printf("\n");
    }
}

File: week 6/sentimental-mario-more/mario.py
from cs50 import get_int

# Get height
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

# Print layers
for i in range(0, height):
    # Space to the left
    for j in range(1, height-i):
        print(" ", end="")

    # Bricks on the left
    for j in range(height-i, height+1):
        print("#", end="")

    # Gap of two
    print("  ", end="")

    # Bricks on the right
    for j in range(height-i, height+1):
        print("#", end="")

    print("")